CODE_STYLE.md

```diff
@@ -126,6 +126,25 @@ Use `re_error::format(err)` when displaying an error.
 
 
 ## C++
+For compatibility reasons we use C++11 (and nothing newer).
+We use `clang-format` to enforce most style choices (see [`.clang-format`](.clang-format)).
+
+### Initialization
+Always use `const` unless you plan on mutating it.
```

Comment:
We probably want to relax this for parameters: While correct and even useful, people usually frown upon `void foo(const int thing)`
(then again, who is to stop us? ;))